libs/core-wrappers/src/lib/Navbar/Navbar.module.css

```diff
@@ -7,32 +7,39 @@
 }
 
 .nav-link {
+  text-decoration: none !important;
   min-width: var(--min-width);
   border: none;
-  padding: 0.5rem 1rem;
   padding-left: var(--horizontal-buffer);
   color: #707070;
   cursor: pointer;
   display: block;
 }
-.nav-link:hover {
-  color: #707070;
+
+.nav-content {
+  height: 34px;
   text-decoration: none;
+  padding-left: 2rem;
 }
 
-.nav-link.active {
-  /* TODO: Use a variable here */
-  background: rgba(#9d85ef, 0.25);
+.nav-content:hover {
+  background: var(--global-color-accent--weak);
+}
+
+.nav-active {
+  background: var(--global-color-accent--weak);
 }
 
 .nav-text {
-  padding-left: 16px;
+  padding-left: 1rem;
+  font-size: 12px;
```

Comment:
**Different Units.**

<details><summary>Why?</summary>

- `padding-left`: I know I had used `rem` for spacing in Portal. But I did chat with Design months ago. Matt expects pixels for spacing, because "Spacing should not scale with font-size change." I've been doing pixels in new code. — [(Confluence) Shared UI - Constants - Units](https://confluence.tacc.utexas.edu/x/dYAZCg)
- `font-size`: I know `px` was much easier for font-size in Portal. But the base font in tup-ui is `10px` so the math for `rem` is easy now: `10px * 1.2rem = 12px` — [(Confluence) Shared UI - Constants - Units](https://confluence.tacc.utexas.edu/x/dYAZCg)
 
</details>

```suggestion
  padding-left: 10px;
  font-size: 1.2rem;
```